src/main.js

```diff
@@ -30,7 +30,13 @@ var main = function() {
   brain.handleNextroom();
   brain.handleSquadmanager();
   brain.handleIncomingTransactions();
+
+<<<<<<< HEAD
+  //Memory.myRooms = _.create(Room.prototype, Memory.myRooms);
+=======
   //Memory.myRooms = _.create(Room.prototype, Memory.myRooms);  //thinking about save all rooms datas : -CPU => +Memory
+>>>>>>> 9191a56... change visualizer render place
+
```

Comment:
This is what happens to mee aswell when I rebase, maybe it has something to do with removing old commit messages, sometimes that seems to break the commit, not sure what causes it yet, meaybe some of you git gurus could shine some light on this? :D